src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -108,6 +109,15 @@ const TimeEntryForm = (props) => {
       .catch((err) => {});
   }, []);
 
+  useEffect(() => {
+    axios
+      .get(`${ApiEndpoint}/tasks/userProfile?members=${userId}`)
+      .then((res) => {
+        setTasks(res?.data || []); 
+      })
+      .catch((err) => {});
```

Comment:
A warning popup here could be good, but it is not necessary.

Reply:
**Addressed.**